locallib.php

```diff
@@ -418,16 +418,19 @@ function save_answerrecording($formdata, $assignment) {
     $file = $fs->get_file($fileinfo['contextid'], $fileinfo['component'], $fileinfo['filearea'],
                           $fileinfo['itemid'], $fileinfo['filepath'], $fileinfo['filename']);
 
-    // Debugging prints for development purposes
-    /*
-    $out = '<audio controls><source src="'.$formdata->audiostring.'"></audio>';
-    $out .= '<br> <b>File URL:</b> '.moodle_url::make_pluginfile_url($file->get_contextid(), $file->get_component(),
+    // Debugging prints for development purposes - now wrapping them up in variables, if they are usable for example as parameteres.
```

Comment:
Saved these debugging lines in variables audioplayer, audioplayerurl and serveresponseprint just in case for @prTopi . Commenting out code is prohibited in style guidelines so I made this.

Reply:
I think that those could be removed and reintroduced in report phase as prod-ready solution. No need for the url, but for the listening capability.